.test/run.sh

```diff
@@ -1,3 +1,6 @@
+# Usage:
+# (Linux) ./.test/run.sh
+# (MacOS) ./.test/run.sh "x86_64-macos"
```

Comment:
What to pass on Windows? "x86_64-win32" didn't work.

Reply:
x86_64-win32 is the correct platform name for Windows. What kind of error did you get? I suppose there might be other problems with running this bash script on windows.

Reply:
hm, it worked now, I think I got 404 the first time I tried it